docs/src/pages/PhoneInputPage/PhoneInputPage.js

```diff
@@ -80,10 +80,10 @@ class StoryContainer extends Component<Props, State> {
         value={this.state.value}
         dialingCode={this.state.dialingCode}
         dialingCodes={[
-          { code: '1', description: '+1 (US)' },
-          { code: '2', description: '+1 (CA)' },
-          { code: '44', description: '+44' },
-          { code: '55', description: '+55' },
+          { code: '1_us', description: '+1 (US)', numberPrefix: '+1' },
+          { code: '1_ca', description: '+1 (CA)', numberPrefix: '+1' },
+          { code: '44_br', description: '+44 (BR)', numberPrefix: '+44' },
```

Comment:
Some people may disagree with BR being used for this instead of UK

Reply:
God I need more coffee
![coffee](https://media3.giphy.com/media/13USqh5fJ2XnIk/giphy.gif)